index.html

```diff
@@ -114,6 +114,7 @@ <h5 class="disoveryName">Title</h5>
     <script type="text/javascript" src="./checkbox.js"></script>
     <script type="text/javascript" src="./venue.js"></script>
     <script type="text/javascript" src="./discovery.js"></script>
+    <script type="text/javascript" src="./mapStyles2.js"></script>
     <script type="text/javascript" src="./script.js"></script>
     <script type="text/javascript" src="./table.js"></script>
 
```

Comment:
It doesnt look like the mapstyles.js was added. Was this on purpose?

Reply:
I thought I would keep 2 map styles for now. Each is different in how it makes the map look. By simply switching the link to 1 or 2 we will get a different style. Eventually we will just decide on 1, but i was a litt unsure which one looked better